python/python/tests/test_arrow.py

```diff
@@ -317,3 +319,22 @@ def test_roundtrip_image_tensor(tmp_path: Path):
     tensor_image_array_2 = tbl2.take(indices).column(2)
 
     assert tensor_image_array_2.type == tensor_image_array.type
+
+
+def test_image_array_pickle(tmp_path: Path, png_uris):
+    # Note: this test will only fail in PyArrow 12.0.0. It was fixed in 13.0.0.
```

Comment:
This wont be tested in CI with pyarrow 12.0?

Reply:
I know. I think it's sufficient for now that we test locally, which I have done. It would be nice to eventually have CI testing with minimal dependency versions.